src/validationinterface.cpp

```diff
@@ -36,10 +37,12 @@ void UnregisterValidationInterface(CValidationInterface* pwalletIn) {
     g_signals.EraseTransaction.disconnect(boost::bind(&CValidationInterface::EraseFromWallet, pwalletIn, _1));
     g_signals.SyncTransaction.disconnect(boost::bind(&CValidationInterface::SyncTransaction, pwalletIn, _1, _2));
     g_signals.UpdatedBlockTip.disconnect(boost::bind(&CValidationInterface::UpdatedBlockTip, pwalletIn, _1));
+    g_signals.SyncBwtCeasing.disconnect(boost::bind(&CValidationInterface::SyncBwtCeasing, pwalletIn, _1, _2));
```

Comment:
For symmetry move this at the beginning; the same for line 45 below

Reply:
Done